deploy/docker/start_backend.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
Let's set the `errexit` option, so that this script exits when one of the commands fail, instead of continuing as if nothing happened, which is the default.

```suggestion
#!/bin/bash

set -o errexit

```